rsciio/utils/image.py

```diff
@@ -41,13 +41,18 @@
 
 def _parse_axes_from_metadata(exif_tags, sizes):
     if exif_tags is None:
-        return []
-    offsets = exif_tags.get("FocalPlaneXYOrigins", [0, 0])
-    # jpg files made with Renishaw have this tag
-    scales = exif_tags.get("FieldOfViewXY", [1, 1])
+        # if no exif_tags exist, axes are set to scale of 1 pixel/pixel
+        # return of axes must not be empty, or dimensions are lost
+        offsets = [0,0]
+        scales = [sizes[1],sizes[0]]
+        unit = "pixel"
```

Comment:
In this case, the standard pattern is to set to None and in hyperspy will be parsed a `traits.api.undefined` value.
```suggestion
        unit = None
```